Makefile.vars.mk

```diff
@@ -28,3 +32,13 @@ KUBENT_DOCKER   ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(root_volume) --entrypoint=/app
 
 instance ?= defaults
 test_instances = tests/defaults.yml
+
+# https://hub.docker.com/r/kindest/node/tags
+KIND_NODE_VERSION ?= v1.24.0
+KIND_IMAGE ?= docker.io/kindest/node:$(KIND_NODE_VERSION)
+KIND_KUBECONFIG ?= $(kind_dir)/kind-kubeconfig-$(KIND_NODE_VERSION)
+KIND_CLUSTER ?= $(PROJECT_NAME)-$(KIND_NODE_VERSION)
```

Comment:
```suggestion
KIND_KUBECONFIG ?= $(kind_dir)/kind-kubeconfig
KIND_CLUSTER ?= $(PROJECT_NAME)
```

Reply:
I left the naming as we have in the providers, do you have smth against that?

Reply:
The name where we append the version number to the cluster name actually originates back to K8up, where we engineered e2e test for multiple K8s versions. Since that we've never had the need to test against multiple K8s version again, but the suffix stayed throughout all the copy-pastas after.
We might start to remove the suffix in the projects, but I have no hard feeling if you want to keep it.